docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+services:
+  proxy:
+    container_name: proxy
+    image: lucaslorentz/caddy-docker-proxy:ci-alpine
+    restart: unless-stopped
+    ports:
+      - 80:80
+      - 443:443
+      - 443:443/udp
+    environment:
+      - CADDY_INGRESS_NETWORKS=extensions_sandbox
+    networks:
+      - sandbox
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./.local/caddy:/data/tls
+
+  cache:
+    container_name: cache
+    image: redis:6
+    restart: unless-stopped
+    networks:
+      - sandbox
+
+  directus:
+    container_name: directus
+    image: directus/directus:10.11.1
```

Comment:
Can we not pin a version? Otherwise we have a constant chore to keep up to date.

Reply:
I only plan to update the version for more critical releases. This version fixed the cookie issue for example. In terms of testing, it's better we are all using the same version as well to stop any "works on my computer" issues.